### DISCUSSION TITLE
Snapshot fails in scrypted/homekit

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#1295] [Q&A]
Not using the api key, in yml I got wb_auth=false, under volumes I set ./img/:/img/, using snapshot=RTSP600. In scrypted, pointing to url http://192.168.0.133:5000/img/front-door.jpg - what am I missing?

**Comment by mrlt8** [comment]
Was it working before, and/or is it still working in a browser?  I don't think you need `snapshot=RTSP600`

Can you try:
```
http://192.168.0.133:5000/snapshot/front-door.jpg
```
or
```
http://192.168.0.133:5000/img/front-door.jpg?exp=600
```

**Reply by cheme75** [reply]
> Was it working before, and/or is it still working in a browser?  I don't think you need `snapshot=RTSP600`
> 
> 
> 
> Can you try:
> 
> ```
> 
> http://192.168.0.133:5000/snapshot/front-door.jpg
> 
> ```
> 
> or
> 
> ```
> 
> http://192.168.0.133:5000/img/front-door.jpg?exp=600
> 
> ```

Not sure when snapshots stopped. I was offline for about 6 months and just got things setup again about 6 wks ago. I think they worked ok at the start. They work fine in the web ui. The web ui looks great all around. Just no snapshots appear in the scrypted ui or in HomeKit. If I tap on the snapshot in either, the stream opens with no issues. Then when I close stream, snapshot times out with snapshot failed. I saw an http 200 error code I think. Pc locked up then charger was in a bad plug so battery died. So waiting for tomorrow before I get back to check it out. Also,before battery died I had just updated to 2.9.11 and started getting connection timeouts for the db3 cam.

**Reply by cheme75** [reply]
I’ll try again tomorrow assuming I can get db cam to connect again. I put the rstp600 just due to not needing frequent snapshot updates. But I’m unclear on how it works so maybe it’s unnecessary.

**Reply by cheme75** [reply]
Ok, updated to .12 which fixed the db cam timeout so great stream again. Neither url will display a snapshot in scrypted hence none in HomeKit. It's not a super critical issue since streaming works fine and I do get people detection in HomeKit so it reduces the nuisance notifications. Just kind irritating I can't figure out why. My only guess is the /img folder on my bridge pc won't allow network access - I'll try sharing that folder. Only other guess is a scrypted issue since fallback should be the prebuffer.

**Reply by cheme75** [reply]
I removed rtsp600 from yml, restarted and still get snapshot failure in scrypted ui, but homekit now shows a normal snapshot updating every 10-12 sec. Kind of odd but it's ok for my needs. It same refresh rate as the blink cams I have in homebridge so maybe it's the HomeKit default.
